### DISCUSSION TITLE
游戏大桶 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#120] [模板讨论]
游戏大桶 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
Failed at 3/6 request,
Fail assert: {"re": "错误", "from": "content"} from failed_asserts,
Request URL: https://gkinto.com/wp-content/themes/modown/action/login.php

**Reply by wjf0214** [reply]
带验证码登录的，OCR识别不是百分百准确，这种一般是验证码识别错误，登录失败了。
多试几次即可。
